features/step_definitions/comments_steps.rb

```diff
@@ -0,0 +1,21 @@
+Given("I have created character and comment") do
+  FactoryBot.create(:character,
+                    :user => @registered_user,
+                    :name => "Wizzard",
+                    :description => "My first wizzard")
+
+  visit user_path(@registered_user)
+  click_on "Wizzard"
+  click_on "New Comment"
+  fill_in "comment_description", :with => "My first comment"
+  click_on "Submit Comment"
+end
```

Comment:
Try to pass Wizzard as variable to the step, lake this:
`There is a character "Wizzard" with one comment` then you can use the "Wizzard" inside step.